benchmarks/targets/hover-brs.js

```diff
@@ -0,0 +1,29 @@
+module.exports = async (suite, name, brighterscript, projectPath, options) => {
```

Comment:
Added a new benchmark target that exemplifies the difference with symbol caching

Reply:
love it!